trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -159,14 +158,13 @@ const submitDealPUT = async (req, res) => {
   const {
     dealId,
     dealType,
-    checker,
   } = req.body;
   let deal;
 
   const canSubmitDealAfterUkefIds = await dealHasAllUkefIds(dealId);
 
   if (canSubmitDealAfterUkefIds) {
-    deal = await submitDealAfterUkefIds(dealId, dealType, checker);
+    deal = await submitDealAfterUkefIds(dealId, dealType);
```

Comment:
`checker` is used in a function in `submitDealAfterUkefIds`, seems this will break some things?

Reply:
good spot! I've reverted this